ocs_ci/utility/utils.py

```diff
@@ -402,6 +402,50 @@ def download_file(url, filename):
     assert r.ok
 
 
+def get_url_content(url):
+    """
+    Return URL content
+
+    Args:
+        url (str): URL address to return
+    Returns:
+        str: Content of URL
+
+    Raises:
+        AssertionError: When couldn't load URL
+
+    """
+    log.debug(f"Download '{url}' content.")
+    r = requests.get(url)
+    assert r.ok, f"Couldn't load URL: {url} content!"
+    return r.content
+
+
+def expose_nightly_ocp_version(version):
```

Comment:
s/expose_nightly_ocp_version/get_nightly_ocp_version

Reply:
The function exposes the version for latests OCP version from given version. So I will stick with origin name if others ppl disagree as well.

Reply:
get aligns with all the get functions we have there, using get_url, exposes seems one off wording in the module, but I see you already merged here!